src/dil_demo/http_utils.clj

```diff
@@ -8,7 +8,7 @@
 (ns dil-demo.http-utils)
 
 (defn parse-www-authenticate
-  "Parse given `WWW-Authenticate` header with expected format:
+  "Parse given `WWW-Authenticate` header with following format.
```

Comment:
ik zou "with following format" eruit halen en dan in volgende regels

"expects  format `auth-scheme key=\"value\" other=\"thing\"`"

of zo doen